db/pre-re/status.yml

```diff
@@ -115,6 +115,13 @@ Body:
       Inspiration: true
       Warmer: true
       Gvg_Freez: true
+      Stone: true
+      StoneWait: true
+      Freeze: true
+      Stun: true
+      Sleep: true
+      Burning: true
+      #Undead: true
     End:
       Dancing: true
```

Comment:
Freeze and Stun make you stop only when you are singing/dancing, similar to Stone. Do we need to apply the special handling for Stone also for these?

And I never tested Whiteimprison in that regard.

Reply:
For White Imprison, I confirmed that it doesn't stop my current walk, only the click-to-walk after White Imprison is active.